### PR TITLE
Upgrade driver & sidecar dependencies for v1.25.0

### PR DESCRIPTION
**Pull request by AndrewSirenko**
**Is this a bug fix or adding new feature?**
Release

**What is this PR about? / Why do we need it?**
Upgrading dependencies ahead of v1.25.0 of aws-ebs-csi-driver. (And small govet-suggested fix in node.go from klog version bump)

**What testing is done?** 
`make test` `make verify`

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml [hunk 1]
```diff
@@ -160,20 +158,19 @@ sidecars:
     securityContext:
       readOnlyRootFilesystem: true
       allowPrivilegeEscalation: false
-
+  snapshotController:
+    image:
+      tag: v6.3.2-eks-1-28-9
```
Comment (torredil) [inline charts/aws-ebs-csi-driver/values.yaml:163]: This parameter is not used anywhere in the chart template.

File: charts/aws-ebs-csi-driver/values.yaml [hunk 2]
```diff
@@ -160,20 +158,19 @@ sidecars:
...
 proxy:
   http_proxy:
   no_proxy:
-
 imagePullSecrets: []
 nameOverride:
 fullnameOverride:
-
```
Comment (ConnorJC3) [inline charts/aws-ebs-csi-driver/values.yaml:171]: Don't remove this line

File: charts/aws-ebs-csi-driver/values.yaml [hunk 3]
```diff
@@ -160,20 +158,19 @@ sidecars:
...
 awsAccessSecret:
   name: aws-secret
   keyId: key_id
   accessKey: access_key
-
```
Comment (ConnorJC3) [inline charts/aws-ebs-csi-driver/values.yaml:176]: Don't remove this line

File: charts/aws-ebs-csi-driver/values.yaml [hunk 4]
```diff
@@ -403,19 +399,17 @@ node:
...
+# See docs/additional-daemonsets.md for more information
+#
+# example:
+#   nodeSelector:
+#     node.kubernetes.io/instance-type: c5.large
+#   volumeAttachLimit: 15
 
 # Enable compatibility for the A1 instance family via use of an AL2-based image in a separate DaemonSet
 # a1CompatibilityDaemonSet: true
-
```
Comment (ConnorJC3) [inline charts/aws-ebs-csi-driver/values.yaml:418]: Don't remove this line

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]
There seems to be some sort of re-formatting applied to `values.yaml` that is removing a lot of lines by your editor or something? Please undo.

File: charts/aws-ebs-csi-driver/values.yaml (hunk 2 above)
Comment (AndrewSirenko) [reply charts/aws-ebs-csi-driver/values.yaml:171]: `yq` in the update-sidecar-dependencies is linting the file. Is there any particular reason why we shouldn't remove the line?

Comment (AndrewSirenko) [reply charts/aws-ebs-csi-driver/values.yaml:171]: `yq` in the release script is linting the yaml files. Is there any particular reason this line shouldn't be removed?

Comment (AndrewSirenko) [reply charts/aws-ebs-csi-driver/values.yaml:171]: Answered offline*

**Review by torredil — APPROVED** [review]
/approve

**Pull request merged**